### DISCUSSION TITLE
Background color and text color for select options

### DISCUSSION THREAD

**Discussion by tyteen4a03** [payloadcms/payload#3916] [Feature Requests & Ideas]
It would be great if we can have built-in support for this!

**Comment by GermanJablo** [comment]
What do you mean with "select options"?

**Reply by tyteen4a03** [reply]
Something like this:

![Screenshot 2024-10-03 at 13 43 40](https://github.com/user-attachments/assets/df422c13-7c50-4abb-8614-9c6581b9be5a)
